packages/components/src/input-control/backdrop.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { memo } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { Fieldset, Legend, LegendText } from './styles/input-control-styles';
+
+function Backdrop( {
```

Comment:
Components should have [JSDoc comments with type info](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#documenting-wordpresselement-react-components).

Reply:
Just to confirm, it sounds like we need to add this to every component, correct?

Looking at previous components, there doesn't appear to be many examples of JSDoc comments:

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/select-control/index.js

I'm guessing the goal is to add these over time to increase doc coverage?

Reply:
When dealing with _private_ components such as Backdrop, for now, I wouldn't let this block development.